homeassistant/components/teslemetry/services.yaml

```diff
@@ -130,3 +130,137 @@ speed_limit:
           min: 1000
           max: 9999
           mode: box
+
+add_charge_schedule:
+  fields:
+    device_id:
+      required: true
+      selector:
+        device:
+          filter:
+            integration: teslemetry
+    days_of_week:
+      required: true
+      selector:
+        select:
+          options:
+            - "[%key:common::time::monday%]"
+            - "[%key:common::time::tuesday%]"
+            - "[%key:common::time::wednesday%]"
+            - "[%key:common::time::thursday%]"
+            - "[%key:common::time::friday%]"
+            - "[%key:common::time::saturday%]"
+            - "[%key:common::time::sunday%]"
```

Comment:
translation references only work in strings.json

use a translation key here

Reply:
How do you translate selector values? I cant find any examples or references on how to do that? https://developers.home-assistant.io/blog/2023/07/11/translating-services/